data_steward/tools/load_vocab.sh

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
```

Comment:
Given that this is a 95 line bash script and only likely to get longer, why is it a bash script?  What is it doing that can't be done in python?